internal/config.go

```diff
@@ -307,6 +315,46 @@ func CheckAllowedSettings(config *viper.Viper) {
 	}
 }
 
+func bindToEnv(k string, val string) error {
+	if err := os.Setenv(strings.ToUpper(k), val); err != nil {
+		return errors.Wrap(err, "Failed to bind config to env variable")
+	}
+	return nil
+}
+
+func AssertRequiredSettingsSet() error {
```

Comment:
Does it mean that cobra and viper doesn't check it?

Reply:
I think yes. I just decomposed the old code into several methods, this is one of them

Reply:
Hmm